src/System.ComponentModel.Composition.Registration/src/Configurations.props

```diff
@@ -1,12 +1,7 @@
 ﻿<Project>
   <PropertyGroup>
-    <PackageConfigurations>
-      netstandard;
-    </PackageConfigurations>
     <BuildConfigurations>
-      $(PackageConfigurations);
-      netcoreapp;
-      uap;
+      netstandard2.1;
```

Comment:
In here you are again dropping support for netstandard compatible frameworks. This is only ok if we have never shipped this package as stable before.